provision/minikube/rebuild.sh

```diff
@@ -26,6 +26,10 @@ helm repo add grafana https://grafana.github.io/helm-charts
 helm install --debug tempo grafana/tempo -n monitoring -f tempo.yaml
 helm install --debug loki grafana/loki -n monitoring -f loki.yaml
 helm install --debug promtail grafana/promtail -n monitoring -f promtail.yaml
+kubectl create namespace keycloak
```

Comment:
it would be nice if those instructions will be shared, something like a separate `install_keycloak.sh` called in `rebuild.sh`, `upgrade.sh` and `Makefile` ?

Reply:
Both `makefile` and `upgrade.sh` are not ideal as they don't really allow a seamless upgrade. I'd love to see `make` to install only what has changed, and to switch between install and upgrade for the help command as necessary. But I'll keep that for another PR.

Reply:
Thanks for explaining!
Maybe naive question, have you tried to use everywhere `helm upgrade --install`?